web/src/lib/components/admin-page/delete-confirm-dialoge.svelte

```diff
@@ -30,9 +36,25 @@
 <ConfirmDialogue title="Delete User" confirmText="Delete" on:confirm={handleDeleteUser} on:cancel>
   <svelte:fragment slot="prompt">
     <div class="flex flex-col gap-4">
-      <p>
-        <b>{user.name}</b>'s account and assets will be permanently deleted after {$serverConfig.userDeleteDelay} days.
-      </p>
+      {#if !forceDelete}
+        <p>
+          <b>{user.name}</b>'s account and assets will be scheduled for permanent deletion in {$serverConfig.userDeleteDelay}
+          days.
+        </p>
+      {:else}
+        <p>
+          <b>{user.name}</b>'s account and assets will be queued for permanent deletion <b>immediately</b>.
+        </p>
+      {/if}
+
+      <div class="flex justify-center m-4 gap-2">
+        <label class="text-sm dark:text-immich-dark-fg" for="forceDelete">
+          Queue user and assets for immediate deletion
+        </label>
+
+        <input type="checkbox" class="form-checkbox h-5 w-5 color" bind:checked={forceDelete} />
```

Comment:
You use `<label for="forceDelete">` above, so you should add that ID to the input field to link them together

```suggestion
        <input id="forceDelete" type="checkbox" class="form-checkbox h-5 w-5 color" bind:checked={forceDelete} />
```

Reply:
I switched from slide to checkbox and didn't add, thanks!